### PR TITLE
I18n(fr) : Update `guides/cms/*` from #9240

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
 Update `guides/cms/*` from #9240

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/cms/keystatic.mdx [hunk 1]
```diff
@@ -137,7 +139,7 @@ export default config({
 ```
 
 :::note[Vous utilisez déjà des collections de contenu ?]
-Si vous utilisez déjà [content collections](/fr/guides/content-collections/#définition-des-collections) dans votre projet Astro, mettez à jour le schéma ci-dessus pour qu'il corresponde exactement à la (aux) collection(s) définie(s) dans votre schéma existant.
+Si vous utilisez déjà [content collections](/fr/guides/content-collections) dans votre projet Astro, mettez à jour le schéma ci-dessus pour qu'il corresponde exactement à la (aux) collection(s) définie(s) dans votre schéma existant.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/keystatic.mdx:142]: fix:
```suggestion
Si vous utilisez déjà [content collections](/fr/guides/content-collections/) dans votre projet Astro, mettez à jour le schéma ci-dessus pour qu'il corresponde exactement à la (aux) collection(s) définie(s) dans votre schéma existant.
```

File: src/content/docs/fr/guides/cms/keystatic.mdx [hunk 2]
```diff
@@ -204,7 +206,7 @@ const posts = await getCollection('posts')
 
 ### Affichage d'un seul article
 
-Pour afficher le contenu d'un article individuel, vous pouvez importer et utiliser le composant `<Content />` pour [rendre votre contenu en HTML](/fr/guides/content-collections/#rendre-le-contenu-en-html) :
+Pour afficher le contenu d'un article individuel, vous pouvez importer et utiliser le composant `<Content />` pour [rendre votre contenu en HTML](/fr/guides/content-collections/#rendre-le-contenu-du-corps) :
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/keystatic.mdx:209]: We should keep the previous anchor in the meantime:
```suggestion
Pour afficher le contenu d'un article individuel, vous pouvez importer et utiliser le composant `<Content />` pour [rendre votre contenu en HTML](/fr/guides/content-collections/#rendre-le-contenu-en-html) :
```
(I updated the section in #10145 and the anchor will be `restitution-du-contenu` but with the broken links... I guess this PR will be merged later)

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Two broken links, but after that it looks good!

**Review by ArmandPhilippot — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
Nice work, LGTM!

**Pull request merged**